src/utils/queries.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const API_URL = '/api/v1';
+
+const getResource = async (_key, id) => {
```

Comment:
Interesting -- this `_key` is required by react query and is `resources` when we call it?

Reply:
yeah, we don't use `_key` at all here but React Query uses it to id cache responses, I believe.
> The unique key you provide is used internally for refetching, caching, deduping related queries.
https://react-query.tanstack.com/docs/guides/queries#query-basics

In our case, how `useQuery` works, it passes the key and a param to the callback (`getResource`) so in order to get the id, I had to accommodate for the first arg `useQuery` passes.

Reply:
Got it; thanks for linking to the docs!